src/app/debug/table/table.component.ts

```diff
@@ -378,6 +381,12 @@ export class TableComponent implements OnInit, OnDestroy {
     });
   }
 
+  openDeleteModal(): void {
+    if (this.tableSettings.reportMetadata.length > 0) {
+      this.deleteModal.open([], true);
```

Comment:
This function is here to delete all reports, so I don't need the reports that have to be deleted, because they are all of them. Maybe I can make this argument optional?

Reply:
Yes that would make more sense imo